cmd/keytransparency-server/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN go get -tags="mysql" ./cmd/keytransparency-server
 ENTRYPOINT /go/bin/keytransparency-server \
 	--addr="$HOST:$RPC_PORT" \
 	--db="${DB_USER}:${DB_PASSWORD}@tcp(${DB_HOST})/${DB_DATABASE}" \
-	--vrf="$VRF_KEY_PATH" \
+	--vrf="$VRF_PRIV" \
```

Comment:
Good catch! I wanted consistent naming with `prepare_server.sh` where this variable is also called `VRF_PRIV`. I missed that I didn't rename the [default](https://github.com/gdbelvin/keytransparency/pull/2/files#diff-0e69bceb3e39e5774b7edcdc1aae92caL12) in this particular Dockerfile, though.